.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,95 @@
+name: Feature
```

Comment:
What do we want to do about feature **requests**? Its not necessarily something that belongs to an Epic, right?